config.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import logging
+
+from lya import AttrDict
+
+logger = logging.getLogger(__name__)
+
+
+class Config(AttrDict):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def load(self):
+        try:
+            self.from_yaml(open('settings.yaml', 'r'))
+        except FileNotFoundError as exc:
+            message = 'File settings.yaml is missing'
+            logger.error(message)
+            raise exc
```

Comment:
Why do you need this ```try``` if you are raising error? Just to log it? Traceback is saying what happens.. It's enough.